src/utility/svd.jl

```diff
@@ -152,20 +218,28 @@ function random_start_vector(t::AbstractMatrix)
 end
 
 # Compute SVD data block-wise using KrylovKit algorithm
-function TensorKit._tsvd!(
-    f, alg::IterSVD, trunc::Union{NoTruncation,TruncationSpace}, p::Real=2
-)
+function TensorKit.tsvd!(
+    f, alg::SVDAdjoint{F}; trunc::Union{NoTruncation,TruncationSpace}=notrunc(), p::Real=2
+) where {F<:IterSVD}
     # early return
     if isempty(blocksectors(f))
         truncerr = zero(real(scalartype(f)))
         return _empty_svdtensors(f)..., truncerr
     end
 
-    SVDdata, dims = _compute_svddata!(f, alg, trunc)
+    SVDdata, dims = _compute_svddata!(f, alg.fwd_alg, trunc)
     U, S, V = _create_svdtensors(f, SVDdata, dims)
-    truncerr = trunc isa NoTruncation ? abs(zero(scalartype(f))) : norm(U * S * V - f, p)
+    truncation_error =
+        trunc isa NoTruncation ? abs(zero(scalartype(f))) : norm(U * S * V - f, p)
 
-    return U, S, V, truncerr
+    # construct info NamedTuple
+    truncation_error /= norm(S)
```

Comment:
I don't think I agree that `tsvd!` should give you a normalized truncation error by default.